packages/admin/src/utils/handleAxiosError.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosError } from 'axios';
+import { getLoginUrl } from './general';
+
+const handleAxiosError = (error: AxiosError) => {
+  console.error('Failed to verify token', error);
+  const unauthorized =
+    error?.response?.status === 401 || error.response?.status === 403;
+  if (unauthorized) {
+    return {
+      redirect: {
+        destination: '/404',
+        permanent: false,
+      },
+    };
+  }
+  return {
+    redirect: {
+      destination: getLoginUrl(),
+      permanent: false,
+    },
+  };
+};
```

Comment:
I think the name here is a bit misleading as this is actually more of a general error handler to run while fetching data, it doesn't just handle axios errors and the error passed doesn't really need to be one. I would just type it to accept an `unknown` argument and give it more general name, and then check the type in the function like this:

```suggestion
const handleRequestError = (error: unknown) => {
  console.error('Failed to verify token', error);
  if (error instanceof AxiosError) {
    const unauthorized =
      error.response?.status === 401 || error.response?.status === 403;
    if (unauthorized) {
      return {
        redirect: {
          destination: '/404',
          permanent: false,
        },
      };
    }
  }
  return {
    redirect: {
      destination: getLoginUrl(),
      permanent: false,
    },
  };
};
```